### PR TITLE
i have created a class to get the data from the database

### PR DESCRIPTION
**Pull request by Olubiggy**

### CODE REVIEW

**Comment by anderscc** [comment]
Hey, @Olubiggy there seem to be some merge conflicts with your branch.

File: GeneticAlgorithm.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Pretty table helps with displaying output, we can use something else if it works better.
 import prettytable as prettytable
+from django.db import models
```
Comment (Caleb7621-java) [inline GeneticAlgorithm.py:3]: We should not be importing the models into the GeneticAlgorithm.py file.

File: GeneticAlgorithm.py [hunk 2]
```diff
@@ -689,6 +691,30 @@ def set_meetingTime(self, meetingTime): self._meetingTime = meetingTime
...
+
+#class GettersModel(models.Model):
+    #GATA = models.CharField()
+    #Course = models.TextField()
+    #Labs = models.TextField()
+    #Assignment = models.TextField()
+    #SemesterYear = models.CharField()
+    #Schedules = models.TextField()
+
+class Getters():
```
Comment (Caleb7621-java) [inline GeneticAlgorithm.py:704]: Is this the best way of getting data from the database? We should be making an API call and selecting data given a specific Semester and Year.

**Review by Caleb7621-java — CHANGES_REQUESTED** [review]
See line comments.

File: GeneticAlgorithm.py [hunk 3]
```diff
@@ -689,6 +691,30 @@ def set_meetingTime(self, meetingTime): self._meetingTime = meetingTime
...
+    def __init__(self, get_gata, get_courses, get_labs):
```
Comment (Caleb7621-java) [inline GeneticAlgorithm.py:705]: All of this should be implemented within the Data class.

**Review by Caleb7621-java — CHANGES_REQUESTED** [review]
Changes should be made within Data class for pulling data into Algorithm.

**Comment by Olubiggy** [comment]
i have done this task again

**Pull request closed without merging**